warp/Network/Wai/Handler/Warp/Types.hs

```diff
@@ -94,6 +94,17 @@ type BufferPool = IORef ByteString
 -- | Type for buffer
 type Buffer = Ptr Word8
 
+-- | A write buffer of a specified size
+-- containing bytes and a way to free the buffer.
+data WriteBuffer = WriteBuffer {
+      bufBytes :: Buffer
```

Comment:
In my opinion, `bufBytes` is confusing. What about `bufBuffer`?

Reply:
I agree. Renamed.